presto-verifier/src/main/java/com/facebook/presto/verifier/framework/DataVerification.java

```diff
@@ -161,7 +169,19 @@ else if (QUERY_BANK_MODE.equals(runningMode)) {
     @Override
     protected DeterminismAnalysisDetails analyzeDeterminism(QueryObjectBundle control, DataMatchResult matchResult)
     {
-        return determinismAnalyzer.analyze(control, matchResult.getControlChecksum());
+        if (isUseTestChecksumInDeterminismAnalyzer) {
+            // If we are told to use test checksum for determinism analysis then helper action points to the test,
```

Comment:
Remove this comment. helper doesn't always point to test, and someone may want to run determinism analysis on test even if they aren't using a helper cluster.  But anyway this comment isn't important, since we explicitly pass test and control actions.